llvm/lib/Analysis/TargetLibraryInfo.cpp

```diff
@@ -367,6 +369,8 @@ static void initializeLibCalls(TargetLibraryInfoImpl &TLI, const Triple &T,
       TLI.setUnavailable(LibFunc_fmaxf);
       TLI.setUnavailable(LibFunc_fmin);
       TLI.setUnavailable(LibFunc_fminf);
+      TLI.setAvailableWithName(LibFunc_hypot, "_hypot");
+      TLI.setAvailableWithName(LibFunc_hypotf, "_hypotf");
```

Comment:
Needs tests with the nonstandard name

Reply:
I have tried to write the test and hope it can pass the CI.

Reply:
I don't think this test shows both of these call names are recognized. It would kind of work if any of the checks showed that _hypotf was emitted from what was originally a _hypot

Reply:
OK. I will remove these nonstandard names first because I need to learn something before adding these nonstandard names.

Here is an off-topic question: Can you give me some suggestions for testing the nonstandard names? (I searched for tests for _copysign and _copysignf and tried to write tests like that, but it seems not good.) Thanks if you can give me some suggestions.

Reply:
I would assume it would be a matter of writing optimization tests like this that use the nonstandard names.

Alternatively, we should expand the llvm-tli-checker tests. Currently it just handles ps4, but we should have baseline tests for all the other platforms

Reply:
Thanks. I will learn something about these first.